code/modules/mob/living/carbon/human/emote.dm

```diff
@@ -355,7 +355,9 @@
 		return
 	if("tail" in dna.species.mutant_bodyparts)
 		dna.species.mutant_bodyparts -= "tail"
+		dna.species.mutant_bodyparts -= "spines"

```

Comment:
I'd move the two spines parts under
``` if("spines" in dna.species.mutant_bodyparts)```

since at the moment you're adding spines whether they have them or not by the looks of things.